boards/arm/s32k1xx/rddrone-bms772/README.txt

```diff
@@ -0,0 +1,142 @@
+README
```

Comment:
Please add the board Documentation too: https://nuttx.apache.org/docs/latest/platforms/index.html

Reply:
Ok! Thank you!